lib/mastodon/emoji_cli.rb

```diff
@@ -41,9 +41,14 @@ def import(path)
 
       Gem::Package::TarReader.new(Zlib::GzipReader.open(path)) do |tar|
         tar.each do |entry|
+          # File exists && Extension is PNG && not a macOS shadow file (begin with ._)
```

Comment:
Comment seems to repeat what's on the line below. The macOS part is relevant but it's not above the line where it's actually checked.